index.rst

```diff
@@ -0,0 +1,23 @@
+.. Mbed TLS documentation master file, created by
+   sphinx-quickstart on Thu Jul 29 17:27:29 2021.
+   You can adapt this file completely to your liking, but it should at least
+   contain the root `toctree` directive.
+
+Welcome to Mbed TLS's documentation!
+====================================
+
+.. toctree::
+   :maxdepth: 2
```

Comment:
(likely non-actionable for now)  A depth of 2 looks about right but I don't think the current structure works. The home page shouldn't have direct links to sections of each subpage. But the subpages should be grouped by topic: review guidelines under contributing, security advisories under news or security, long-term plans and maintainers under project, ...

But it's a bit hard to anticipate the right structure when we have so little content. We can go with what we have for now and restructure it as we go.